### PR TITLE
Add new blog post adopted from internal Coinbase blog

### PR DESCRIPTION
**Pull request by ernieturner**
This new post is written in MDX format so I took advantage of a few Docusaurus features to add an image component which makes embedding images a bit better. 

I'm not entirely sure why the `Cargo.lock` file needs to be updated, but added it to this PR to try and get things to pass.

### CODE REVIEW

File: website/blog/2021-12-08-introducing-the-new-relay-compiler.mdx [hunk 1]
```diff
@@ -61,15 +63,17 @@ And while we're not there yet, we have made significant achievements on each of
 
 For example, the new compiler ships with support for the new `@required` directive, which will nullify the parent linked field or throw an error if a given subfield is null when read out. This may sound like a trivial quality-of-life improvement, but if half of your component's code is null checks, `@required` starts to look pretty good!
 
-<style type="text/css">{" img { max-height: 600px; }"}</style>
+:::note A component without `@required`
```
Comment (ernieturner) [inline website/blog/2021-12-08-introducing-the-new-relay-compiler.mdx:66]: I thought wrapping these images in admonitions looked a bit better than the current block quote

File: website/blog/2023-01-03-resilient-relay-apps.mdx [hunk 2]
```diff
@@ -0,0 +1,357 @@
...
+hide_table_of_contents: false
+---
+
+import Image from '@site/components/Image';
+
+:::tip Guest Post
+
+This is a guest post written by Ernie Turner, a Staff Engineer at Coinbase. Coinbase has thoroughly adopted Relay in their applications and is a strong ally of the Relay Team. Last year they helped co-develop the [Relay VSCode extension](https://marketplace.visualstudio.com/items?itemName=meta.relay). Ernie has agreed to share this internal enginnering blog post with us.
+
+:::
```
Comment (ernieturner) [inline website/blog/2023-01-03-resilient-relay-apps.mdx:14]: I wasn't really sure what to write here, so definitely give me some feedback on this. I'm open to changing anything here.

**Comment by davidmccabe** [comment]
Thanks Ernie!!! Taking a look through it.

@alunyov Do you know what's going on with the Cargo file?

**Comment by tbezman** [comment]
nice @ernieturner

**Pull request closed without merging**